### DISCUSSION TITLE
How to connect to a CMS??

### DISCUSSION THREAD

**Discussion by theroboteenager** [adityatelange/hugo-PaperMod#1059] [Q&A]
Hello I am building a Hugo website using this theme and I was wondering how to connect it to an external CMS ??

Is it possible with this theme ??

Any help would be highly appreciated.

**Comment by RoneoOrg** [comment]
Another user apparently [added Netlify CMS](https://github.com/DarthBenro008/blogs/commit/6b8b78cc2a09c7ce9d5502a4a6392435bd64504b) to Papermod

Note that the future of Netlify CMS is [not clear right now](https://answers.netlify.com/t/is-this-project-dead/70988).

Please report your findings here, I'm interested in Netlify CMS too
